### DISCUSSION TITLE
Connection to OpenVPN on other host

### DISCUSSION THREAD

**Discussion by 2003flo** [bnhf/openvpn-admin-plus#65] [Q&A]
Hello,
I am already using openvpn-admin-plus successfully on a system that also runs OpenVPN/Pivpn.

I have now acquired a new system: I have created virtual machines on the new system:

OpenVPN/Pivpn is installed in virtual machine A.

Openvpn-admin-plus and all other Docker containers are running in another virtual machine B.

Now my question:
Is it possible to connect openvpn-admin-plus from virtual machine B to virtual machine A? Or do both have to run on the same host?

**Comment by karabelnikov** [comment]
Hi. No, the web interface is associated with the installation of OpenVPN and must be installed on the same host.
